src/remoteplayer.h

```diff
@@ -32,6 +32,16 @@ enum RemotePlayerChatResult
 	RPLAYER_CHATRESULT_KICK,
 };
 
+struct CloudSettings
```

Comment:
including clouds.h in remoteplayer means you expose client part to server, it's not acceptable. cloudsettings.h sounds reasonable